atomix/cluster/src/main/java/io/atomix/raft/cluster/impl/RaftClusterContext.java

```diff
@@ -124,6 +125,7 @@ public DefaultRaftMember getMember(final MemberId id) {
     raft.getThreadContext()
         .execute(
             () -> {
+              MDC.put("partitionId", Integer.toString(raft.getPartitionId()));
```

Comment:
:question: Why put this in the cluster context and not the Raft context?

Reply:
Not sure whether I get the question. This is the first time the thread of the raft server is executed, which is why I choosed it. The partition id is part of the raft context as the thread context, which we use here?

Reply:
What I mean is, why not do that where you create the thread? It's unexpected for me here, and I'm not sure it's guaranteed that this will always be the first time anything is executed on that thread. Why not simply enqueue a job on the thread immediately after its creation to set the MDC for it?

Reply:
ok; wasn't sure whether this would make more sense but fine for me